contracts/registries/LicenseRegistry.sol

```diff
@@ -238,9 +238,13 @@ contract LicenseRegistry is ILicenseRegistry, AccessManagedUpgradeable, UUPSUpgr
 
         for (uint256 i = 0; i < parentIpIds.length; i++) {
             _verifyDerivativeFromParent(parentIpIds[i], childIpId, licenseTemplate, licenseTermsIds[i]);
-            $.parentIps[childIpId].add(parentIpIds[i]);
             $.childIps[parentIpIds[i]].add(childIpId);
-            $.attachedLicenseTerms[childIpId].add(licenseTermsIds[i]);
+            // determine if duplicate license terms
```

Comment:
do you need to check the return value of line 241/242?

Reply:
We don't need to, as when registering a derivative to parents, we only check the duplication of license terms of parents.